app.rb

```diff
@@ -0,0 +1,166 @@
+require_relative 'person'
+require_relative 'student'
+require_relative 'book'
+require_relative 'classroom'
+require_relative 'teacher'
+require_relative 'rental'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def start
+    puts 'Welcome to our School Library!'
+    loop do
+      menu
+      option = gets.chomp
+      break if option == '7'
+
+      get_num option
+    end
+    puts 'Thank you for using our Library!'
+  end
+
+  def menu
+    puts
+    puts 'Please choose an option by entering a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts '7 - Exit'
+  end
```

Comment:
- [ ] this is exactly the same code as you are using in the main.rb file which is not in line with the DRY rules, please make sure that you don't write pieces of code twice unnecessarily.